neutopia/src/chest.rs

```diff
@@ -21,6 +21,57 @@ impl Chest {
 
         Ok(())
     }
+
```

Comment:
Can you split out the chest name support into a separate PR?

Reply:
@konkers done